tests/components/transmission/test_config_flow.py

```diff
@@ -100,21 +102,18 @@ def init_config_flow(hass):
     assert result["data"][CONF_PORT] == PORT
 
     # test with all provided
-    result = await flow.async_step_user(MOCK_ENTRY)
+    result = await hass.config_entries.flow.async_init(
+        transmission.DOMAIN, context={"source": "user"}, data=MOCK_ENTRY
+    )
 
-    assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-    assert result["title"] == NAME
-    assert result["data"][CONF_NAME] == NAME
-    assert result["data"][CONF_HOST] == HOST
-    assert result["data"][CONF_USERNAME] == USERNAME
-    assert result["data"][CONF_PASSWORD] == PASSWORD
-    assert result["data"][CONF_PORT] == PORT
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
```

Comment:
We should split this test into three pieces. Each test should just init the flow once, unless we want to test how two simultaneous flows work.

We should not change the assertions here but still test the same things.